polkadot-protocol/architecture/system-chains/asset-hub.md

```diff
@@ -1,4 +1,204 @@
+
+[API Sidecar](https://github.com/paritytech/substrate-api-sidecar){target=\_blank} is a RESTful service that can be deployed alongside Polkadot and Kusama nodes. It provides endpoints to retrieve real-time blockchain data, including asset information. For Asset Hub, Sidecar allows querying:
+
+- **Asset look-ups** - retrieve specific assets using `AssetId`
+- **Asset balances** - view the balance of a particular asset on Asset Hub
+
+Public instances of API Sidecar connected to Asset Hub are available, such as:
+
+- [Polkadot Asset Hub Sidecar](https://polkadot-asset-hub-public-sidecar.parity-chains.parity.io/){target=\_blank}
+- [Kusama Asset Hub Sidecar](https://kusama-asset-hub-public-sidecar.parity-chains.parity.io/){target=\_blank}
+
+These public instances are primarily for ad-hoc testing and quick checks.
+
+### TxWrapper
+
+[`TxWrapper`](https://github.com/paritytech/txwrapper-core){target=\_blank} is a library that simplifies constructing and signing transactions for Polkadot SDK-based chains, including Polkadot and Kusama. This tool includes support for working with Asset Hub, enabling developers to:
+
+- Construct offline transactions
+- Leverage asset-specific functions such as minting, burning, and transferring assets
+
+TxWrapper provides the flexibility needed to integrate asset operations into custom applications while maintaining the security and efficiency of Polkadot's transaction model.
+
+### Asset Transfer API
+
+[Asset Transfer API](https://github.com/paritytech/asset-transfer-api){target=\_blank} is a library focused on simplifying the construction of asset transfers for Polkadot SDK-based chains that involve system parachains like Asset Hub. It exposes a reduced set of methods that facilitate users sending transfers to other parachains or locally. Refer to the [cross-chain support table](https://github.com/paritytech/asset-transfer-api/tree/main#current-cross-chain-support){target=\_blank} for the current status of cross-chain support development.
+
+Key features include:
+
+- Support for cross-chain transfers between parachains
+- Streamlined transaction construction with support for the necessary parachain metadata
+
+The API supports various asset operations, such as paying transaction fees with non-native tokens and managing asset liquidity.
+
+### Parachain Node
+
+To fully leverage the Asset Hub's functionality, developers will need to run a system parachain node. Setting up an Asset Hub node allows users to interact with the parachain in real time, syncing data and participating in the broader Polkadot ecosystem. Guidelines for setting up an [Asset Hub node](https://github.com/paritytech/polkadot-sdk/tree/master/cumulus#asset-hub-){target=\_blank} are available in the Parity documentation.
+
+Using these integration tools, developers can manage assets seamlessly and integrate Asset Hub functionality into their applications, leveraging Polkadot's powerful infrastructure.
+
+## XCM Transfer Monitoring
```

Comment:
This feels like it should be in another page, but in the future maybe we can consider moving out as it's out of scope for this PR.